src/swap/exolix.ts

```diff
@@ -173,7 +206,7 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
       }
     } else {
       const nativeMin = await request.toWallet.denominationToNative(
-        rateResponse.min_amount,
+        rateResponse.minAmount.toString(),
```

Comment:
you already check minimums above. This logic now looks incorrect for 'to' quotes assuming minAmount is always the 'from' amount

Reply:
@paullinator Thanks for noticing, corrected.

Reply:
@paullinator Thanks for noticing, corrected.